modules/persistent-storage-byok.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-azure.adoc
+// storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+// storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_content-type: CONCEPT
+[id="byok_{context}"]
+= About "bring your own key" encryption
+
+The "Bring your own key" (BYOK) feature encrypts root volumes and the default storage class.
```

Comment:
For this `encrypts root volumes and the default storage class.` and  `The installation program also encrypts the default storage class with this key` in Line 13:
Here I think it doesn't encrypt the storageclass, but encrypt the PersistentVolume (created with pre-created storageclass by CSI Driver Operator)